src/Raven.Studio/wwwroot/App/views/database/query/query.html

```diff
@@ -260,9 +260,9 @@ <h2 class="on-base-background">
             <div class="control-label">Cache enabled</div>
         </div>
         <div class="col-xs-5">
-            <div class="toggle">
+            <div class="toggle flex-row">
```

Comment:
I can see you started using flex. Great. I hope flexbox froggy was helpful. 

![image](https://user-images.githubusercontent.com/1250431/74857098-7b041200-5343-11ea-8921-375193ca35c6.png)